package.json

```diff
@@ -20,7 +20,7 @@
     "snyk": "^1.364.0",
     "ts-loader": "5.4.5",
     "tslint": "5.5.0",
-    "typescript": "3.7.5",
+    "typescript": "3.8.3",
```

Comment:
The latest `jest-editor-support@28.0.1` version  requires typescript 3.8+ https://github.com/babel/babel/issues/10981